README.md

```diff
@@ -28,94 +28,17 @@ Decidim is a participatory democracy framework written on Ruby on Rails original
 
 ## What do you need to do?
 
+- [Get started with Decidim](#getting-started-with-decidim)
```

Comment:
this should be "Getting started with Decidim" also (the link and the title should be the same)